### PR TITLE
add avatar and badge component

### PR DESCRIPTION
**Pull request by sk-ip**
please review avatar and badge component

### CODE REVIEW

File: docs/avatar/index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+          <img src="https://picsum.photos/200/200" class="avatar" />
+          <img src="https://picsum.photos/200/200" class="avatar avatar-sm" />
+          <img src="https://picsum.photos/200/200" class="avatar avatar-md" />
+          <img src="https://picsum.photos/200/200" class="avatar avatar-lg" />
+        </div>
+        <iframe
+          src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=material&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522https%253A%252F%252Fpicsum.photos%252F200%252F200%2522%2520class%253D%2522avatar%2522%2520%252F%253E%250A%253Cimg%2520src%253D%2522https%253A%252F%252Fpicsum.photos%252F200%252F200%2522%2520class%253D%2522avatar%2520avatar-sm%2522%2520%252F%253E%250A%253Cimg%2520src%253D%2522https%253A%252F%252Fpicsum.photos%252F200%252F200%2522%2520class%253D%2522avatar%2520avatar-md%2522%2520%252F%253E%250A%253Cimg%2520src%253D%2522https%253A%252F%252Fpicsum.photos%252F200%252F200%2522%2520class%253D%2522avatar%2520avatar-lg%2522%2520%252F%253E"
```
Comment (nyyshaaa) [inline docs/avatar/index.html]: should give alt tags for images

**Review by nyyshaaa — APPROVED** [review]

File: component/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji",
+    "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+}
+
+/* Avatar */
+
+.avatar {
+    border-radius: 50%;
+    width: 32px;
+    height: 32px;
```
Comment (Aditi0505) [inline component/avatar/avatar.css]: Can convert values > 10px to rem.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. Also, agree with @nightlight-wander suggestions. 
Rest looks good.

File: docs/avatar/index.html (hunk 1 above)
Comment (sk-ip) [reply docs/avatar/index.html]: thanks

File: component/avatar/avatar.css (hunk 2 above)
Comment (sk-ip) [reply component/avatar/avatar.css]: thanks have done the changes in all the avatar classes

**Pull request merged**